modules/stochastic_tools/src/state_sim/StateProcessor.C

```diff
@@ -0,0 +1,75 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+#include "StateProcessor.h"
+#include <stdlib.h>
+#include <string>
+#include <cmath>
+//#include "StateSimBase.h"
+
+using namespace std;
+
+StateProcessor::StateProcessor(int max_time_step)
+  : _max_time_step(max_time_step),
+    _ev_times()
+{
+
+  //add a bunch of random time items
+  int r = rand() % 10;
```

Comment:
Using rand() is bad idea. We'll need to use Moose's random number generator if we want this to pass testing.